third_party/js.bzl

```diff
@@ -212,8 +212,8 @@ def tensorboard_js_workspace():
       licenses = ["notice"],  # Apache License 2.0
       sha256_urls = {
           "85a2ff924f1bf4757976aca22fd0efb045d9b3854f5a4ae838c64e4d11e75005": [
```

Comment:
This PR doesn’t update the file hash, so Bazel will still read the old
version of the code if it’s in your local cache, or fail with a fetch
error otherwise. Please update the checksum to the new value.

Testing that the projector still works, as you described, is
great—thanks! Is there a way that we can check that the new features
were properly integrated? What are the changes in this version bump?

Reply:
In particular, this PR introduces the following error on a clean cache
(note the checksum mismatch message):

```
INFO: Call stack for the definition of repository 'ai_google_pair_umap_js' which is a filegroup_external (rule definition at /HOMEDIR/.cache/bazel/_bazel_wchargin/52a95bbdd50941251730eb33b7476a66/external/io_bazel_rules_closure/closure/filegroup_external.bzl:186:22):
 - /HOMEDIR/git/tensorboard/third_party/js.bzl:209:3
 - /HOMEDIR/git/tensorboard/third_party/workspace.bzl:35:3
 - /HOMEDIR/git/tensorboard/WORKSPACE:98:1
WARNING: Download from http://mirror.tensorflow.org/unpkg.com/umap-js@1.2.2/lib/umap-js.min.js failed: class com.google.devtools.build.lib.bazel.repository.downloader.UnrecoverableHttpException Checksum was 035fede477f10b909dd64a2ea01c031149ee523f54fb9bbe48a170eb04d53825 but wanted 85a2ff924f1bf4757976aca22fd0efb045d9b3854f5a4ae838c64e4d11e75005
WARNING: Download from https://unpkg.com/umap-js@1.2.2/lib/umap-js.min.js failed: class com.google.devtools.build.lib.bazel.repository.downloader.UnrecoverableHttpException Checksum was 035fede477f10b909dd64a2ea01c031149ee523f54fb9bbe48a170eb04d53825 but wanted 85a2ff924f1bf4757976aca22fd0efb045d9b3854f5a4ae838c64e4d11e75005
ERROR: An error occurred during the fetch of repository 'ai_google_pair_umap_js':
   java.io.IOException: Error downloading [http://mirror.tensorflow.org/unpkg.com/umap-js@1.2.2/lib/umap-js.min.js, https://unpkg.com/umap-js@1.2.2/lib/umap-js.min.js] to /HOMEDIR/.cache/bazel/_bazel_wchargin/52a95bbdd50941251730eb33b7476a66/external/ai_google_pair_umap_js/umap-js.min.js: Checksum was 035fede477f10b909dd64a2ea01c031149ee523f54fb9bbe48a170eb04d53825 but wanted 85a2ff924f1bf4757976aca22fd0efb045d9b3854f5a4ae838c64e4d11e75005
ERROR: /HOMEDIR/git/tensorboard/tensorboard/components/tf_imports/BUILD:89:1: no such package '@ai_google_pair_umap_js//': java.io.IOException: Error downloading [http://mirror.tensorflow.org/unpkg.com/umap-js@1.2.2/lib/umap-js.min.js, https://unpkg.com/umap-js@1.2.2/lib/umap-js.min.js] to /HOMEDIR/.cache/bazel/_bazel_wchargin/52a95bbdd50941251730eb33b7476a66/external/ai_google_pair_umap_js/umap-js.min.js: Checksum was 035fede477f10b909dd64a2ea01c031149ee523f54fb9bbe48a170eb04d53825 but wanted 85a2ff924f1bf4757976aca22fd0efb045d9b3854f5a4ae838c64e4d11e75005 and referenced by '//tensorboard/components/tf_imports:umap-js'
ERROR: Evaluation of query "deps(//tensorboard)" failed: errors were encountered while computing transitive closure
```

Reply:
Ok, updated the hash and it seems to work from a clean cache now.

Re: testing the integrated features-- the main change is to project new points (umap.transform()), which I manually tested in the chrome dev tools since the embedding projector doesn't use this feature normally.

The one problem now, though, is that [umap-js.d.ts](https://github.com/tensorflow/tensorboard/blob/master/tensorboard/plugins/projector/vz_projector/umap.d.ts) in tensorboard doesn't match the new minified file. It looks like the umap declaration file is copied from [this](https://raw.githubusercontent.com/PAIR-code/umap-js/1.0.3/lib/umap-js.d.ts) file in the umap repo, but the corresponding 1.2.2 version of that file doesn't seem to exist, and neither yarn build nor yarn bundle generated it. Any thoughts? I can also bother Andy about it (it's his TODO :) )

Reply:
Hmm, good catch. Looks like this was deleted in this PR:
<https://github.com/PAIR-code/umap-js/pull/3>

Yes, let’s bother Andy :-). @cannoneyed, was this removal intentional?
It doesn’t look like there are any `.d.ts` files left in the repository.
What was the intended migration plan?

Reply:
So I was chatting with @cannoneyed about this on Friday, and he said that umap-js.d.ts was actually manually created and was just for the specific methods that were used in the embedding projector (Andy, let me know if I'm misunderstanding.)

Given that none of those methods are changing, maybe we don't have to update umap-js.d.ts?

Reply:
I think Andy's now out on paternity leave-- @wchargin, do you have thoughts on what to do here?